scripts/index.js

```diff
@@ -1 +1,28 @@
-// add your solutions here
\ No newline at end of file
+// add your solutions here
+arrayOne = [1,2,3,4,5,6,7,8,9,10,-1,-2,-3,20,99,97,200, 220, 12];
+
+function evenNumbers(numbers) {
+    let evenArray = numbers;
```

Comment:
Since assigning a variable to an array just adds a pointer (instead of making a copy), would it be possible to leave out the `evenArray` variable?